Library/Formula/ghc.rb

```diff
@@ -1,10 +1,10 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.1/ghc-7.10.1-src.tar.xz"
-  sha256 "92f3e3d67a637c587c49b61c704a670953509eb4b17a93c0c2ac153da4cd3aa0"
+  url "https://downloads.haskell.org/~ghc/7.10.2/ghc-7.10.2-src.tar.xz"
+  sha256 "54cd73755b784d78e2f13d5eb161bfa38d3efee9e8a56f7eb6cd9f2d6e2615f5"
 
-  revision 1
+  revision 0
```

Comment:
Sorry, I wasn't very clear.  `revision 0` is the default, so this line can be omitted altogether.

(Pushing another change will trigger a new build, though, so you can wait to fix this until we see if anything else needs changing.)